Tests/SystemInfo/Collector/JsonComposerLockSystemInfoCollectorTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * File containing the JsonComposerLockSystemInfoCollectorTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\Tests\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\JsonComposerLockSystemInfoCollector;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value\ComposerPackage;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value\ComposerSystemInfo;
+use PHPUnit_Framework_TestCase;
+
+class JsonComposerLockSystemInfoCollectorTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var JsonComposerLockSystemInfoCollector
+     */
+    private $composerCollector;
+
+    /**
+     * @var JsonComposerLockSystemInfoCollector
+     */
+    private $composerCollectorNotFound;
+
+    public function setUp()
+    {
+        $this->composerCollector = new JsonComposerLockSystemInfoCollector(__DIR__ . '/_fixtures/composer.lock');
+        $this->composerCollectorNotFound = new JsonComposerLockSystemInfoCollector(__DIR__ . '/_fixtures/snafu.lock');
```

Comment:
I wouldn't have bothered with creating private vars, given that they're only used once each.